### DISCUSSION TITLE
removing the numbers in parentesis for artists / labels

### DISCUSSION THREAD

**Discussion by VinylVault** [joalla/discogs_client#56] [Support] [answered]
I know this is possibly out of scope for the module, but is there a way, to remove the (1), (13) etc after labels, artists etc?

Personally, I would enable this feature by default, seeing as all artists, labels etc have a unique ID

**Comment by JOJ0** [comment, marked as answer]
> I know this is possibly out of scope for the module, but is there a way, to remove the (1), (13) etc after labels, artists etc?

since the parentheses are coming from discogs, I 'd also see this as being out of scope, but if you'd file a PR that let users **optionally** remove them using a smart regex we could consider it

> 
> 
> Personally, I would enable this feature by default, seeing as all artists, labels etc have a unique ID

default I'd say definitely no to it since one of the next users would complain that discogs_client is not giving the same results as discogs web search does ;-)
